tbx/core/wagtail_hooks.py

```diff
@@ -18,7 +18,7 @@
 @hooks.register("before_serve_document", order=100)
 def serve_document_from_s3(document, request):
     # Skip this hook if not using django-storages boto3 backend.
-    if not issubclass(default_storage, S3Storage):
```

Comment:
Nit: This feels like `isinstance` with extra steps

Reply:
highlighted that above https://github.com/torchbox/torchbox.com/pull/344#discussion_r1918684298

will convert